doc/content/gettingstarted/play.md

```diff
@@ -0,0 +1,265 @@
+
+>Note: if you do not specify a priority, the default priority `LOWEST` is used. You can specify `LOWEST`, `LOW`, `BELOW_NORMAL`, `NORMAL`, `ABOVE_NORMAL`, `HIGH` and `HIGHEST`.
+
+The stack supports some cool features, such as confirmed downlink with your own correlation IDs. For example, you can push this:
+
+```json
+{
+  "downlinks": [{
+    "f_port": 15,
+    "frm_payload": "vu8=",
+    "priority": "HIGH",
+    "confirmed": true,
+    "correlation_ids": ["my-correlation-id"]
+  }]
+}
+```
+
+Once the downlink gets acknowledged, a message is published to the topic `v3/{application id}/devices/{device id}/down/ack`:
+
+```json
+{
+  "end_device_ids": {
+    "device_id": "dev1",
+    "application_ids": {
+      "application_id": "app1"
+    },
+    "dev_eui": "4200000000000000",
+    "join_eui": "4200000000000000",
+    "dev_addr": "00E6F42A"
+  },
+  "correlation_ids": [
+    "my-correlation-id",
+    "..."
+  ],
+  "downlink_ack": {
+    "session_key_id": "AWnj0318qrtJ7kbudd8Vmw==",
+    "f_port": 15,
+    "f_cnt": 11,
+    "frm_payload": "vu8=",
+    "confirmed": true,
+    "priority": "NORMAL",
+    "correlation_ids": [
+      "my-correlation-id",
+      "..."
+    ]
+  }
+}
+```
+
+Here you see the correlation ID `my-correlation-id` of your downlink message. You can add multiple custom correlation IDs, for example to reference events or identifiers of your application.
+
+#### Class C unicast
+
+In order to send class C downlink messages to a single device, enable class C support for the end device using the following command:
+
+```bash
+$ ttn-lw-cli end-devices update app1 dev1 --supports-class-c
+```
+
+This will enable the class C downlink scheduling of the device. That's it! New downlink messages are now scheduled as soon as possible.
+
+To disable class C scheduling, set reset with `--supports-class-c=false`.
+
+>Note: you can also pass `--supports-class-c` when creating the device. Class C scheduling will be enable after the first uplink message which confirms the device session.
+
+#### Class C multicast
+
+Multicast messages are downlinks messages which are sent to multiple devices that share the same security context. In the Network Server, this is an ABP session. See [creating a device](#createdev) for learning how to create an ABP device.
+
+Multicast sessions do not allow uplink. Therefore, you need to explicitly specify the gateway(s) to send messages from, using the `class_b_c` field:
+
+```json
+{
+  "downlinks": [{
+    "f_port": 15,
+    "frm_payload": "vu8=",
+    "priority": "NORMAL",
+    "class_b_c": {
+      "gateways": [{
+        "gateway_ids": {
+          "gateway_id": "gtw1"
+        }
+      }]
+    }
+  }]
+}
+```
+
+>Note: if you specify multiple gateways, the Network Server will try the gateways in the order specified. The first gateway with no conflicts and no duty-cycle limitation will send the message.
```

Comment:
> The first gateway with no conflicts and no duty-cycle limitation will send the message.

Doesn't sound too professional.
IMO this should be part of the first sentence and contain something like "respect regional duty-cycle regulations"